lerobot/configs/env/dora_aloha_real.yaml

```diff
@@ -2,6 +2,12 @@
 
 fps: 30
 
+eval:
```

Comment:
I'm not sure this is relevant here as this is not simulation.

Reply:
Aha. My fault! will delete it

so dora is for real robotics only